<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrayStarterPage</class>
 <widget class="QWidget" name="TrayStarterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>473</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Tray Starter&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Icon</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="standardButton">
        <property name="toolTip">
         <string>Select to use the standard icon</string>
        </property>
        <property name="text">
         <string>Standard Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highContrastButton">
        <property name="toolTip">
         <string>Select to use the high contrast icon</string>
        </property>
        <property name="text">
         <string>High Contrast Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="blackWhiteButton">
        <property name="toolTip">
         <string>Select to use a black and white icon</string>
        </property>
        <property name="text">
         <string>Black and White Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="blackWhiteInverseButton">
        <property name="toolTip">
         <string>Select to use an inverse black and white icon</string>
        </property>
        <property name="text">
         <string>Inverse Black and White Icon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>464</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
